Makefile

```
@@ -330,29 +217,39 @@ release: build
 debug: BUILD_MODE := Debug
 debug: build
 
-test: check
+test: check qjs-links
```

checkmake (0.3.2)

[warning] Target body for "test" exceeds allowed length of 5 lines (7).

(maxbodylength)